### PR TITLE
{geo} WRF CrayIntel 2015.11-XC

### PR DESCRIPTION
**Pull request by gppezzi**

### CODE REVIEW

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
dora105 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/2d09ac4669fb869d351c for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**FAILED**
Build succeeded for 5 out of 6 (4 easyconfigs in this PR)
swan - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/5c16e1029fc0263f57e5 for a full test report.

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in this PR)
swan - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/88c99f962101f1bfcd9a for a full test report.

**Comment by gppezzi** [comment]
First build on swan failed because I tried to compile in parallel the ncurses eb (from PR2606). 

@boegel But why do I get 4 builds on dora against 6 on swan? is it due to the presence of existing modules?

**Comment by gppezzi** [comment]
@boegel any remarks here?

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name = 'ncurses'
+version = '5.9'
+
+homepage = 'http://www.gnu.org/software/ncurses/'
+description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
+ and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
+ function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2015.11-XC'}
+toolchainopts = {'optarch': True, 'shared': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: `shared` is enabled by default, so is `optarch`?

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['ncurses-%(version)s_configure_darwin.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb]: a bit weird to have this here (this patch is specific to OS X)
 Add a line note

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
daint104 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/9783ddb7b888f522d0a5 for a full test report.

**Comment by boegel** [comment]
@gppezzi: good to go after remarks are fixed imho...

**Comment by gppezzi** [comment]
@boegel i've removed ncurses from this PR since it was already fixed on #2606

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
dora105 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/6758aedf7148e0506c41 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks @gppezzi!

**Pull request merged**